lib/link.js

```diff
@@ -11,8 +11,16 @@ export default class Link extends Component {
 
   static propTypes = {
     children: PropTypes.oneOfType([
-      PropTypes.string,
-      PropTypes.element
+      PropTypes.element,
+      (props, propName) => {
```

Comment:
I'm seems bit weird to me. This is the first time, I have seen this.
Basically, we should not surprise people who read the code. 

If I was doing this, I'll make this `PropTypes.node` and do the checking on the `render()` function.